Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM haproxy:1.7-alpine
+FROM haproxy:2.3-alpine
 
 RUN apk update && apk --no-cache add bash
 
-ENV DOCKER_GEN_VERSION 0.7.3
+ENV DOCKER_GEN_VERSION 0.7.6
 
 ADD https://github.com/jwilder/docker-gen/releases/download/$DOCKER_GEN_VERSION/docker-gen-linux-amd64-$DOCKER_GEN_VERSION.tar.gz /tmp/docker-gen.tar.gz
```

Comment:
for M1 chip, also need to swap from amd64 to arm64 so it runs correctly. see #11  for my workaround.